### DISCUSSION TITLE
Install Deming with Portainer

### DISCUSSION THREAD

**Discussion by jlgarnier** [dbarzin/deming#83] [Q&A]
Hi,

I have my own virtual server and several applications running on it which I manage with Portainer. I can't use the Dockerfile_web as I fear this would break the other stacks. Can I just install Deming using the docker-compose.yml file, as a new Portainer stack?

Thanks in advance for any help!

**Comment by dbarzin** [comment]
I have limited familiarity with Portainer and minimal experience with Docker. Unfortunately, I am unable to provide assistance in this area.

**Reply by jlgarnier** [reply]
Thanks Didier, I hope someone will jump in and provide some hints on the topic!
